stl/inc/limits

```diff
@@ -1013,20 +1015,83 @@ public:
 };
 
 #ifdef __cpp_lib_bitops // TRANSITION, VSO-1020212
+
+inline bool _Is_tzcnt_supported() {
+    // tzcnt and lzcnt support are indicated with
+    // different cpuid flags.
+    int cpuinfo[4];
+    __cpuid(cpuinfo, 0x07);

```

Comment:
Ditto "this won't compile on non-Intel." Ditto "move into the architecture conditional" ;)